DigitalLearningSolutions.Web/Views/TrackingSystem/Shared/_NavMenuItems.cshtml

```diff
@@ -0,0 +1,32 @@
+﻿@using DigitalLearningSolutions.Web.Helpers
+@using Microsoft.Extensions.Configuration
+@inject IConfiguration Configuration
+
+@if (User.HasClaim(c => c.Type == CustomClaimTypes.UserUserAdmin)) {

```

Comment:
Does this mean that if the user is not super admin, they won't be able to see any tabs? 🤔

Reply:
er possibly...